build.yaml

```diff
@@ -2455,3 +2455,7 @@ node_modules:
   - src/node/ext/server.cc
   - src/node/ext/server_credentials.cc
   - src/node/ext/timeval.cc
+python_dependencies:
```

Comment:
Do we really want this at the top? All of its dependencies are currently above it, which feels about right. Maybe later we move it and `node_modules` in between `libs` and `targets`?